### PR TITLE
Copy fonts with gitignore

### PR DESCRIPTION
**Pull request by noliveleger**
Docker does not delete `jsapp/fonts` any more on build or start.
Instead `rsync` is used to sync `$FONTS_DIR` and `$KPI_SRC_DIR/jsapp/fonts`

### CODE REVIEW

**Comment by jnm** [comment]
Hey @noliveleger, I think it'd be better not to have to maintain the `.gitignore` in two places:
* https://github.com/kobotoolbox/kpi/blob/master/jsapp/fonts/.gitignore
* https://github.com/kobotoolbox/kpi/pull/1956/files#diff-bb08580b36dbd00f817cdbd8ee241935R14

If I don't have time to look at this today, though, I'll go ahead and merge this PR to help make day-to-day development a less painful experience :)

**Comment by noliveleger** [comment]
@jnm, I did it in purpose with Python because I was mistaken. I'm gonna updated the dockerfile, no problemo :-)

**Comment by magicznyleszek** [comment]
@noliveleger John has a good point - what if you'd just move the `jsapp/fonts/.gitignore` rules to the root file?

**Comment by noliveleger** [comment]
@magicznyleszek That was my first intention but because you didn't do it at the first time, I thought there was a reason for that. It seems it doesn't matter, so that will be the solution.
In that case, you will be able to review this PR ;-)

**Comment by magicznyleszek** [comment]
@noliveleger `jsapp/fonts/.gitignore` was already there before my changes, I left it without giving a thought :)

File: .gitignore [hunk 1]
```diff
@@ -23,4 +23,16 @@ test/compiled/*
...
+jsapp/fonts/*.ttf
+jsapp/fonts/*.woff
+jsapp/fonts/*.woff2
+jsapp/fonts/*.scss
+jsapp/fonts/*.css
+jsapp/fonts/*.md
+jsapp/fonts/*.ijmap
+jsapp/fonts/*.otf
+jsapp/fonts/codepoints
+jsapp/fonts/*.html
```
Comment (magicznyleszek) [inline .gitignore]: Make sure you delete `jsapp/fonts/.gitignore` :)

Comment (noliveleger) [reply .gitignore]: Give me the time to do it ;-)

**Comment by noliveleger** [comment]
@magicznyleszek: Files are reverted back and .gitignore is modified. You can start to review it.

**Comment by jnm** [comment]
Success!

**Pull request merged**